src/openllm/utils/import_utils.py

```diff
@@ -334,6 +344,8 @@ def require_backends(o: t.Any, backends: t.MutableSequence[str]) -> None:
     if "tf" not in backends and is_tf_available() and not is_vllm_available(): raise ImportError(VLLM_IMPORT_ERROR_WITH_TF.format(name))
     if "flax" not in backends and is_flax_available() and not is_vllm_available(): raise ImportError(VLLM_IMPORT_ERROR_WITH_FLAX.format(name))
 
+  # Not sure, but do we need to see the backend mapping here?
+  
```

Comment:
We need an autoclass for this.

Reply:
I see which means another `modelling_ggml_auto` inside `models/auto`?

Reply:
yes

Reply:
Cool, also do let me know am I going to the right direction after reviewing rest of the changes.